src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/PrisonApi.java

```diff
@@ -83,14 +83,14 @@ public void putAttendanceForMultipleBookings(final Set<BookingActivity> bookingA
     }
 
     public Page<OffenderAttendance> getAttendanceForOffender(final String offenderNo, final LocalDate fromDate, final LocalDate toDate,
-                                                             final Optional<String> outcome, final Pageable pageable) {
+                                                             final String outcome, final Pageable pageable) {
```

Comment:
Bad style to have Optional as a param apparently

Reply:
i was confused by a java file amongst the kotlin!